modules/installation-configuration-parameters.adoc

```diff
@@ -1000,7 +1000,11 @@ Additional {rh-virtualization} configuration parameters for machine pools are de
 |Number
 
 |`<machine-pool>.platform.ovirt.vmType`
-|Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.
+|Optional. The VM workload type, such as `high-performance`, `server`, or `desktop`.  By default, master nodes use `high-performance`, and worker nodes use `server`. For details, see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Virtual_Machine_General_settings_explained[Explanation of Settings in the New Virtual Machine and Edit Virtual Machine Windows] and link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_High_Performance_Virtual_Machines_Templates_and_Pools[Configuring High Performance Virtual Machines, Templates, and Pools] in the _Virtual Machine Management Guide_.
+[NOTE]
+====
+`high_performance` improves performance on the VM, but there are limitations. For example, you cannot access the VM with a graphical console. For more information see link:https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.4/html-single/virtual_machine_management_guide/index#Configuring_High_Performance_Virtual_Machines_Templates_and_Pools[Configuring High Performance Virtual Machines, Templates, and Pools] in the _Virtual Machine Management Guide_.
```

Comment:
another nit that you can disregard: Consider ", but selecting `high_performance` has limitations" or maybe ", but this `vmType` has limitations."

Reply:
@mikemckiernan I also was not satisfied with the current construction, I played around a bit with it. I think in spite of its apparent awkwardness, it is more clear and concise as is..